tests/unittests/resources/customer_func_path/HttpTrigger/__init__.py

```diff
@@ -0,0 +1,9 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import os
+import azure.functions as func  # NoQA
```

Comment:
Just so that I get this - The `azure.functions as func` here is not the installed `azure.functions` module but a functions module that the customer brings in (or expects). Correct?

Reply:
Yes, this is to mock that the module is loaded from customer's dependencies